docs/src/main/asciidoc/kafka-support.adoc

```diff
@@ -1,11 +1,150 @@
+
+** Prefix: `spring.cloud.stream.kafka.binder.producer-properties.azure`
+** Example: `spring.cloud.stream.kafka.binder.producer-properties.azure`.credential.*
+
+* Spring Cloud Stream Kafka Binder admin configuration
+
+** Prefix: Not supported, should use Spring Boot Kafka common or admin configuration.
+
+.Spring Boot Kafka common configuration options
+[cols="2*", options="header"]
+|===
+|Property
+|Description
+| spring.kafka.properties.azure.credential.client-certificate-password                                         | Password of the certificate file.
+| spring.kafka.properties.azure.credential.client-certificate-path                                             | Path of a PEM certificate file to use when performing service principal authentication with Azure.
+| spring.kafka.properties.azure.credential.client-id                                                           | Client ID to use when performing service principal authentication with Azure. This is a legacy property.
+| spring.kafka.properties.azure.credential.client-secret                                                       | Client secret to use when performing service principal authentication with Azure. This is a legacy property.
+| spring.kafka.properties.azure.credential.managed-identity-enabled                                            | Whether to enable managed identity to authenticate with Azure. If *true* and the `client-id` is set, will use the client ID as user assigned managed identity client ID. The default value is *false*.
+| spring.kafka.properties.azure.credential.password                                                            | Password to use when performing username/password authentication with Azure.
+| spring.kafka.properties.azure.credential.username                                                            | Username to use when performing username/password authentication with Azure.
+| spring.kafka.properties.azure.profile.environment.active-directory-endpoint                                  | The Azure Active Directory endpoint to connect to.
+| spring.kafka.properties.azure.profile.tenant-id                                                              | Tenant ID for Azure resources.
+|===
+
+[NOTE]
+====
+The configuration options in different levels apply the following rules. The more specific configuration options have higher priority than the common ones. For example:
+
+ - Spring Kafka common configuration options supersede the global options.
+ - Spring Kafka consumer configuration options supersede the common options.
+ - Spring Kafka producer configuration options supersede the common options.
+ - Spring Kafka admin configuration options supersede the common options.
+ - The Spring Cloud Stream Kafka Binder options are just like the above.
+====
+
+==== Configurable properties when using Kafka support with plain connection string authentication
+
+
+.Spring Boot Kafka common configuration options
+[cols="2*", options="header"]
```

Comment:
The table header should be `Spring Boot Event Hubs for Kafka common configuration options`